lib/validator-compiler.js

```diff
@@ -1,19 +1,14 @@
 'use strict'
 
-const Ajv = require('ajv').default
-const AjvJTD = require('ajv/dist/jtd')
-
 const defaultAjvOptions = require('./default-ajv-options')
 
 class ValidatorCompiler {
   constructor (externalSchemas, options) {
     // This instance of Ajv is private
     // it should not be customized or used
-    if (options.mode === 'JTD') {
-      this.ajv = new AjvJTD(Object.assign({}, defaultAjvOptions, options.customOptions))
-    } else {
-      this.ajv = new Ajv(Object.assign({}, defaultAjvOptions, options.customOptions))
-    }
+    const ajvPath = ['JTD', '2019', '2020'].includes(options.mode) ? `ajv/dist/${options.mode.toLowerCase()}` : 'ajv'
+    const Ajv = require(ajvPath)
```

Comment:
It doesn't look like same as before. Previously, accessing the `.default`.

```suggestion
    const Ajv = require(ajvPath)
```

Reply:
This allows using Ajv2019 and Ajv2020 by passing 2019 or 2020 in `options.mode`. 

If `options.mode` is not one of `JTD`, `2019`, or `2020`, `ajvPath` is "ajv" (the default).

If I'm misunderstanding the question, please help me understand.

Reply:
I means `const Ajv = require('ajv').default`, now becomes `const Ajv = require('ajv')`